subsquid/scripts/reset-db.sh

```diff
@@ -1,8 +1,8 @@
 set -e
 npx sqd codegen
 npm run build
-rm -rf db/migrations/*.js
+#rm -rf db/migrations/*.js
 npx sqd db drop
 npx sqd db create
-npx sqd db create-migration Init
+#npx sqd db create-migration Init
```

Comment:
Do we want to keep this commented?

Reply:
yeah this should not be called generally on reset